Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -661,7 +661,9 @@ partial class PythonInteractiveEvaluator :
         }
 
         internal Task InvokeAsync(Action action) {
-            return ((System.Windows.UIElement)_window.GetTextViewHost()).Dispatcher.InvokeAsync(action).Task;
+            // TOOD: Verify this
```

Comment:
It's called by DisplayImage and DisplayXaml, and I can see it working for image:
![image](https://cloud.githubusercontent.com/assets/1696845/23001908/7796248c-f39c-11e6-99a6-3f5a386524e7.png)